samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobVideoActionRecognitionSample.java

```diff
@@ -0,0 +1,96 @@
+import com.google.cloud.aiplatform.v1beta1.JobServiceClient;
+import com.google.cloud.aiplatform.v1beta1.JobServiceSettings;
+import com.google.cloud.aiplatform.v1beta1.LocationName;
+import com.google.gson.JsonObject;
+import com.google.protobuf.Value;
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+
+public class CreateBatchPredictionJobVideoActionRecognitionSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "PROJECT";
+    String displayName = "DISPLAY_NAME";
+    String model = "MODEL";
+    String gcsSourceUri = "GCS_SOURCE_URI";
+    String gcsDestinationOutputUriPrefix = "GCS_DESTINATION_OUTPUT_URI_PREFIX";
+    createBatchPredictionJobVideoActionRecognitionSample(
+        project, displayName, model, gcsSourceUri, gcsDestinationOutputUriPrefix);
+  }
+
+  static void createBatchPredictionJobVideoActionRecognitionSample(
+      String project,
+      String displayName,
+      String model,
+      String gcsSourceUri,
+      String gcsDestinationOutputUriPrefix)
+      throws IOException {
+    JobServiceSettings settings =
+        JobServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+    String location = "us-central1";
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (JobServiceClient client = JobServiceClient.create(settings)) {
+      JsonObject jsonModelParameters = new JsonObject();
+      jsonModelParameters.addProperty("confidenceThreshold", 0.5);
+      Value.Builder modelParametersBuilder = Value.newBuilder();
+      JsonFormat.parser().merge(jsonModelParameters.toString(), modelParametersBuilder);
+      Value modelParameters = modelParametersBuilder.build();
+      GcsSource gcsSource = GcsSource.newBuilder().addUris(gcsSourceUri).build();
+      BatchPredictionJob.InputConfig inputConfig =
+          BatchPredictionJob.InputConfig.newBuilder()
+              .setInstancesFormat("jsonl")
+              .setGcsSource(gcsSource)
+              .build();
+      GcsDestination gcsDestination =
+          GcsDestination.newBuilder().setOutputUriPrefix(gcsDestinationOutputUriPrefix).build();
+      BatchPredictionJob.OutputConfig outputConfig =
+          BatchPredictionJob.OutputConfig.newBuilder()
+              .setPredictionsFormat("jsonl")
+              .setGcsDestination(gcsDestination)
+              .build();
+      BatchPredictionJob batchPredictionJob =
+          BatchPredictionJob.newBuilder()
+              .setDisplayName(displayName)
+              // Format: 'projects/{project}/locations/{location}/models/{model_id}'
```

Comment:
For any resource name, we should probably demonstrate the resource name helpers:

```java
String model = com.google.cloud.aiplatform.v1beta1.ModelName.of("project", "location", "model").toString();
```

Reply:
Done.